File: src/main/java/org/kite9/diagram/common/algorithms/det/DetHashSet.java
package org.kite9.diagram.common.algorithms.det;

import java.util.Collection;
import java.util.HashSet;

public class DetHashSet<K extends Deterministic> extends HashSet<K>{

	public DetHashSet() {
		super();
	}

	public DetHashSet(Collection<? extends K> c) {
		super(c);
	}

	public DetHashSet(int initialCapacity) {
		super(initialCapacity);
	}

	
}

File: visualization/kite9-visualization-common/src/main/kotlin/org/kite9/diagram/common/algorithms/det/DetHashSet.kt
package org.kite9.diagram.common.algorithms.det

import kotlin.collections.HashSet

class DetHashSet<K : Deterministic> : HashSet<K> {

    constructor() : super() {}
    constructor(c: Collection<K>) : super(c) {}
    constructor(initialCapacity: Int) : super(initialCapacity) {}
}